kyuubi-server/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,1193 @@
+kyuubi-server
+Copyright 2021-2022 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+@angular/animations 13.1.3 : MIT License
```

Comment:
Should this part belong to LICENCE instead of NOTICE? How do you get or generate this NOTICE file?

Reply:
It's actually a NOTICE of binary release, and maybe we should exclude it from the source release.

Reply:
Here we meet an issue: we put the notice file into a submodule's resource directory. If we need to merge the information into the root NOTICE/LICENSE or not.